adapters/speechbrain/classification/dataloop.json

```diff
@@ -0,0 +1,184 @@
+{
+  "name": "encoderclassifier",
+  "displayName": "Language Classifier",
+  "version": "0.1.17",
+  "scope": "public",
+  "description": "SpeechBrain application for audio language classification.",
+  "attributes": {
+    "Provider": "SpeechBrain",
+    "Deployed By": "Dataloop",
+    "License": "Apache 2.0",
+    "Category": "Model",
+    "Audio classification": "Classification",
+    "Media Type": [
+      "Audio"
+    ],
+    "Libraries": "Torch"
+  },
+  "codebase": {
+    "type": "git",
+    "gitUrl": "https://github.com/dataloop-ai-apps/torch-models.git",
+    "gitTag": "0.1.16"
+  },
+  "components": {
+    "computeConfigs": [
+      {
+        "name": "encoderclassifier-deploy",
+        "runtime": {
+          "podType": "regular-xs",
+          "concurrency": 1,
+          "runnerImage": "gcr.io/viewo-g/piper/agent/gpu/torch-models:0.1.1",
```

Comment:
I know we're going to create a new docker image, but will it be a cpu or gpu one? The podtype and runnerimage contradict each other here.